lib/ansible/plugins/callback/syslog_json.py

```diff
@@ -14,25 +14,25 @@
     version_added: "1.9"
     description:
       - This plugin logs ansible-playbook and ansible runs to a syslog server in JSON format
-      - Before 2.4 only environment variables were available for configuration
+      - Before 2.9 only environment variables were available for configuration
     options:
-      server:
+      syslog_server:
```

Comment:
you don't need to change the option names

Reply:
Thanks. I'll set them back to their original values.